v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent INSERT, UPDATE, UPSERT, and DELETE statements within a single transaction. 
+
+To execute statements in parallel, use the `RETURNING NOTHING` clause for the independent SQL statements.
+
+<div id="toc"></div>
+
+## Independent SQL statements
+
+CockroachDB considers SQL statements to be independent if they operate on independent tables and their execution can be safely reordered without affecting their results. 
+
+For example, the following statements are considered independent since they operate on two independent tables and reordering the statements will not affect the results:
+
+~~~ sql
+> INSERT INTO a VALUES (100);
+> INSERT INTO b VALUES (100);
+~~~
+
+The following statements are not considered independent since they operate on the same table:
+
+~~~ sql
+> INSERT INTO a VALUES (100);
+> INSERT INTO a VALUES (200);
+~~~
+
+The following pairs of statements are dependent since reordering them will affect their results:
+
+~~~ sql
+> UPDATE a SET b = 2 WHERE y = 1;
+> UPDATE a SET b = 3 WHERE y = 1;
+~~~
+
+~~~ sql
+> UPDATE a SET y = true  WHERE y = false;
+> UPDATE a SET y = false WHERE y = true;
+~~~
+
+## Required Privileges
+
+The user must have the `INSERT`, `UPDATE`, `DELETE`, and `UPSERT` [privileges](privileges.html) on the table.
+
+
+## Examples
+
+### Executing independent SQL statements in parallel
+
+~~~ sql
+> SELECT * FROM accounts;
+~~~
+
+~~~
++----+---------+----------+
+| id | balance | customer |
++----+---------+----------+
+|  1 |  1000.0 | Ilya     |
+|  2 |  2000.0 | Nitin    |
+|  3 |  3000.0 | Pam      |
++----+---------+----------+
+(3 rows)
+~~~
+
+~~~ sql
+> BEGIN; 
+> INSERT INTO accounts(id, balance, customer) VALUES (4, 3782, 'Heer') RETURNING NOTHING; 
+> INSERT INTO accounts(id, balance, customer) VALUES (5, 9887, 'Julio') RETURNING NOTHING; 
+> UPDATE accounts SET balance = 1099 WHERE id = 2 RETURNING NOTHING;
+> COMMIT;
+~~~
+
+~~~ sql
+SELECT * FROM accounts;
+~~~
+
+~~~
++----+---------+----------+
+| id | balance | customer |
++----+---------+----------+
+|  1 |  1000.0 | Ilya     |
+|  2 |    1099 | Nitin    |
+|  3 |  3000.0 | Pam      |
+|  4 |    3782 | Heer     |
+|  5 |    9887 | Julio    |
++----+---------+----------+
+(5 rows)
+~~~
+
+The execution timeline for the parallel execution of the independent statements is:
+
+BEGIN &nbsp; &nbsp; &nbsp; INSERT-----\INSERT &nbsp; &nbsp; &nbsp; COMMIT
+<p>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; INSERT-----\INSERT </p>
+<p>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; UPDATE-----\UPDATE </p>
+
+
+### Handling independent and dependent SQL statements  
```

Comment:
I don't think these sections should be nested under the examples header. @jseldess, what do you think?